### PR TITLE
Add design

### PR DESCRIPTION
**Pull request by BiancaMe**
-Add design 
-Create Routes

### CODE REVIEW

File: src/style/navbar.css [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
+  color: rgb(202, 198, 198);
   font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
 }
 
 .item:hover {
-  color: rgb(255, 255, 255);
+  color: rgb(113, 113, 113);
 }
 
 .active {
```
Comment (thecodechaser) [inline src/style/navbar.css:20]: - [ ] Please make sure to add a border between the navbar menu options to follow the wireframe design.

### Your design

image

### Wireframe design

image

File: src/style/calculator.css [hunk 2]
```diff
@@ -5,19 +5,26 @@
...
+.app h2 {
+  font-size: 2.5em;
+  font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+  color: rgb(97, 95, 95);
 }
 
 .calculator {
-  width: 16em;
-  height: 18em;
-  border: 4px solid rgb(251, 251, 251);
```
Comment (thecodechaser) [inline src/style/calculator.css]: - [ ] Please make sure the heading and calculator are horizontally aligned (heading on the left side and calculator on the right side) to follow the wireframe design.

### Your design

image

### Wireframe design

image

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @BiancaMe👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- Good job with the routes. ✔️
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️

- [x] You have disabled linter errors in `App.js, Buttons.js, and navbar.js` files, Please don't disable linter errors instead fix them manually. It will help you to write more clean and professional code.

- [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/style/calculator.css (hunk 2 above)
Comment (BiancaMe) [reply src/style/calculator.css]: Hello, @thecodechaser . You told me I must fix some eslint errors in my files without disabling linter errors. I'm still having problems with the Buttons.js file. 
Buttons.js : 
image

The only way that I found to solve this is disabled the error. If you read this: https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/3d1d26d89d492947cbf69f439deec9e7cfaf9867/docs/rules/no-static-element-interactions.md. 
You can see that : 
image

I think that it is my case. What do you think? Do you have another solution?

File: src/App.css
Comment (joskalenda) [inline src/App.css:1]: - [ ] Your last commit does not contain the changes regarding what follows  as mentioned by the previous reviewer, please make the necessary changes and you will be good to go

 

- [ ] WHAT IS ASKED



- [ ] YOUR DESIGN

**Review by joskalenda — CHANGES_REQUESTED** [review]
| STATUS: REQUIRED CHANGES ♻️  |
|--------------------------------|
| Hi @BiancaMe , |
|Great work on making the changes requested by a previous reviewer 👏, You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet. |
|  |

## Highlights

- [ ].. ✅ Nice design
- [ ].. ✅ Clean code
- [ ]..✅ Green linters

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

-------
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me(@joskalenda) in your question so I can receive the notification.

> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Lizdev-05 — APPROVED** [review]
Hi @BiancaMe, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

:heavy_check_mark: Professional Readme file :+1: 
:heavy_check_mark: Good commit history :+1:
:heavy_check_mark: No linter error :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**